.bumpversion.cfg

```diff
@@ -3,7 +3,7 @@ commit = True
 message = Bump version {current_version} to {new_version}
 tag = False
 tag_name = {new_version}
-current_version = 2.5.1-alpha
+current_version = 2.5.2-alpha
```

Comment:
For the moment you don't have to bump the sardana version, but update the changelog is a must.